content/en/docs/concepts/architecture/nodes.md

```diff
@@ -173,19 +173,15 @@ to be unreachable. (The default timeouts are 40s to start reporting
 ConditionUnknown and 5m after that to start evicting pods.) The node controller
 checks the state of each node every `--node-monitor-period` seconds.
 
-In versions of Kubernetes prior to 1.13, NodeStatus is the heartbeat from the
-node. Starting from Kubernetes 1.13, node lease feature is introduced as an
-alpha feature (feature gate `NodeLease`,
-[KEP-0009](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/0009-node-heartbeat.md)).
-When node lease feature is enabled, each node has an associated `Lease` object in
-`kube-node-lease` namespace that is renewed by the node periodically, and both
-NodeStatus and node lease are treated as heartbeats from the node. Node leases
-are renewed frequently while NodeStatus is reported from node to master only
-when there is some change or enough time has passed (default is 1 minute, which
-is longer than the default timeout of 40 seconds for unreachable nodes). Since
-node lease is much more lightweight than NodeStatus, this feature makes node
-heartbeat significantly cheaper from both scalability and performance
-perspectives.
+Each node has an associated `Lease` object in `kube-node-lease` namespace.
+It is periodically renewed by Kubelet and both NodeStatus and node lease
+are treated as heartbeats from the node.
+Node leases are renewed frequently while NodeStatus is reported from node to
+master only where there is some change or enough time has passed (default is
+5 minutes, which is longer than the default timeout of 40 seconds for
+unreachable nodes). Since Lease is much more lighweigh object than Node,
```

Comment:
1. Typo: “lighweigh”
2. This page is about Nodes, and the reader could be totally new to Kubernetes. This page is a lot of detail to explain what a Node is. If anything, I'd prefer to slim it further.

Reply:
How about a heading: “Node leases” plus 100 words on the control plane issuing Node leases to keep track of Node health / failures.

Does NodeStatus even need mentioning here?

Reply:
I think it does - the reason is that for the last 5 years, NodeStatus was the only signal for heartbeat. Now this is only treated as an additional one.

Reply:
That said, I tried to rephrase it a bit (applied suggestions by @kbhawkey ) and slightly better organize.